### ISSUE TITLE
Incorrectly marking of doi-access=free

### ISSUE THREAD

**Issue by daniel-barrows** [5j9/citer#62]
Over the last 6 months or so, I have noticed that Citer sometimes incorrectly adds `doi-access=free` to articles that are not freely accessible:

## Examples
* https://citer.toolforge.org/?user_input=10.1111%2Fcaje.12237
* https://citer.toolforge.org/?user_input=10.1007%2Fs001480050116&input_type=url-doi-isbn
* https://citer.toolforge.org/?user_input=10.1016%2Fj.jue.2013.07.003
* https://citer.toolforge.org/?user_input=10.1016/j.ejpoleco.2007.06.006

**Comment by 5j9** [comment]
Right, in recent months Citer has been using https://openaccessbutton.org/ to see of an article is openly available or not (thanks to contribution of @EArmour).
For example, in case of [10.1111/caje.12237](https://doi.org/10.1111/caje.12237), openaccessbutton returns https://ageconsearch.umn.edu/record/274657/files/qed_wp_1331.pdf as a freely available version.

I see how Citer's output is confusing since the result does not actually point to the open-access URL.

I just deployed a patch which makes citer include the open-access URL. This might override the original URL in some cases in an undesired manner; I will try to fix such cases if they get reported.

**Comment by daniel-barrows** [comment]
@5j9 While adding a URL is _very_ helpful, the DOI is still marked as doi-access=free, which in my recent experience, is wrong about half the time. I'd suggest removing this parameter until you have a more reliable source for it. I think it's better to omit it when it is actually free than include it when access is actually restricted.

**Comment by 5j9** [comment]
@daniel-barrows Could you provide examples of some of the incorrect cases? In the examples you mentioned in your original report, it seems to point to a correct free-access URL.
